src/main/docker/configs/fester.properties.default

```diff
@@ -42,3 +42,10 @@ DEFAULT_IMAGE_THUMBNAIL_SIZE=
 
 # The version of Festerize that is compatible with this version of Fester
 FESTERIZE_VERSION=
+
+# Optional: Ship logs via Fluency to fluentd endpoint
+FLUENCY_ENABLED=false
```

Comment:
Just a note for history, that we want to get rid of these config files and use ENVs (at runtime) and POM properties (at build time). No change needed now though.